SRM 325/RGBStreet.cpp

```diff
@@ -0,0 +1,192 @@
+#include <cstdio>
+#include <cmath>
+#include <cstring>
+#include <ctime>
+#include <iostream>
+#include <algorithm>
+#include <set>
+#include <vector>
+#include <sstream>
+#include <typeinfo>
+#include <fstream>
+
+using namespace std;
+
+int dp[100][100];
```

Comment:
this could be dp[100][4] as the colour has only 4 possible values: empty, red, green, blue